magprime/automated_emails.py

```diff
@@ -109,10 +109,6 @@ def __init__(self, event):
                          a.gets_any_kind_of_shirt, when=before(c.SHIRT_DEADLINE),
                ident='magprime_shirt_size_confirmation')
 
-AutomatedEmail(Attendee, 'MAGFest Dealer waitlist has been exhausted', 'dealer_waitlist_exhausted.txt',
```

Comment:
This is crazy town. I'd love to see the script that generated the notes that match this.